mods/default/README.txt

```diff
@@ -339,6 +328,36 @@ sonictechtonic (CC BY 3.0):
 https://www.freesound.org/people/sonictechtonic/sounds/241872/
   player_damage.ogg
 
+Sheyvan (CC0 1.0):
+https://freesound.org/people/Sheyvan/sounds/476113/
+  default_dig_choppy.ogg
+
+lolamadeus (CC0 1.0):
+https://freesound.org/people/lolamadeus/sounds/179341/
+  default_gravel_dig.ogg
+  default_gravel_dug.ogg
+
+Benboncan (CC BY 3.0):
+https://freesound.org/people/Benboncan/sounds/71823/
+  default_dig_cracky.ogg
+
+Erdie (CC BY 3.0):
+https://freesound.org/people/Erdie/sounds/41579/
+  default_hard_footstep.ogg
+https://freesound.org/people/Erdie/sounds/41580/
+  default_wood_footstep.ogg
+
+Utsuru (CC0 1.0):
+https://freesound.org/people/utsuru/sounds/183453/
+  default_wood_dug.ogg
+
+tomi_slav (CC0 1.0):
+https://freesound.org/people/tomi_slav/sounds/465811/
+  default_stone_dug.ogg
+
+Zbyhek (CC BY 3.0):
+https://freesound.org/people/Zbyhek/sounds/167489/
+  default_grass_footstep.ogg
```

Comment:
For all the above:
Multiple sounds should be credited in this form:
`default_metal_footstep.*.ogg`

Reply:
Will do!

Reply:
Done